AEPMessaging/Sources/Message+FullscreenMessageDelegate.swift

```diff
@@ -66,6 +68,7 @@ extension Message: FullscreenMessageDelegate {
             // handle optional tracking
             if let interaction = queryParams[MessagingConstants.IAM.HTML.INTERACTION], !interaction.isEmpty {
                 message?.track(interaction, withEdgeEventType: .interact)
```

Comment:
Should probably check for `autoTrack` here and not send the tracking event?

Reply:
i don't view this as an auto track - this is an explicit request to track provided by the URL.

happy to discuss though if you think it's confusing or you see it differently.